RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGenerator.h

```diff
@@ -38,8 +37,8 @@ class CAHitQuadrupletGenerator : public HitQuadrupletGenerator {
 
 public:
 
-    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector&& iC, bool needSeedingLayerSetsHits=true): CAHitQuadrupletGenerator(cfg, iC, needSeedingLayerSetsHits) {}
-    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector& iC, bool needSeedingLayerSetsHits=true);
+    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector&& iC): CAHitQuadrupletGenerator(cfg, iC) {}
+    CAHitQuadrupletGenerator(const edm::ParameterSet& cfg, edm::ConsumesCollector& iC);
 
     virtual ~CAHitQuadrupletGenerator();
```

Comment:
To be more clear: CAHitQuadrupletGenerator does not derive from anything anymore and there is no derived class from it either. Why do we need a virtual destructor in this case?